src/foam/nanos/script/ScriptRunnerDAO.java

```diff
@@ -8,29 +8,74 @@
 
 import foam.core.*;
 import foam.dao.*;
-import java.io.IOException;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+import foam.nanos.pool.FixedThreadPool;
 
 public class ScriptRunnerDAO
   extends ProxyDAO
-{
+  {
+  final private int DEFAULT_WAIT_TIME = 2000;
```

Comment:
No need to make private. Better to be public since it's immutable. Also, it should be static.